packages/lpa-questionnaire-web-app/src/controllers/development-plan.js

```diff
@@ -0,0 +1,24 @@
+const { VIEW } = require('../lib/views');
+const getAppealSideBarDetails = require('../lib/appeal-sidebar-details');
+
+exports.getDevelopmentPlan = (req, res) => {
+  const developmentPlanSection =
+    req.session.appealReply.optionalDocumentsSection.developmentOrNeighbourhood;
```

Comment:
What happens if this doesn't exist? (By the way I'm just typing comments, they're not meant to be terse, just wizzing through it.)

Reply:
ie that's a very deep object reference so it relies on 5 objects existing.

Reply:
It may be the API just handles it and returns 500 if we're happy with that, but ideally, if the params are wrong then 400 I guess maybe?

Reply:
Passes the tests, and that object is set explicitly

Reply:
I suppose my point is we're assuming this route has only the web application as the consumer, which is MVP I guess.

Reply:
I think Marc has a point here. It is something I regret not adding in the earlier section. Simple optional chaining would help, with some defaults when destructuring. If the model changes mid review then it will cause errors with how the data is structured and we don't want that